targets/default.nix

```diff
@@ -8,6 +8,10 @@
     system = "x86_64-linux";
     modules = [
       microvm.nixosModules.host
+      # .microvm for vm-format "host" to nest vms for system development on x86 (Intel)
+      # NOTE: Ghaf nested virtualization is not assumed nor tested yet on AMD
+      microvm.nixosModules.microvm
```

Comment:
By my understanding, the microvm-module should be only imported on the guest-microvm itself